<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Source Sink</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>110</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>110</height>
        </size>
       </property>
       <property name="title">
        <string>Wave Selector</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="source_source_radio">
            <property name="text">
             <string>Source</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="source_xlating_radio">
            <property name="text">
             <string>Xlating</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="source_interpolator_radio">
            <property name="text">
             <string>Interpolator</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
